CHANGELOG.md

```diff
@@ -1,25 +1,14 @@
-## repo-template
+## terraform-aws-tardigrade-resource-group
 
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
-### 1.1.0
+### [1.1.0]
+(https://github.com/plus3it/terraform-aws-tardigrade-resource-group/releases/tag/1.1.0)
```

Comment:
Initial releases should be 1.0.0, and the link should be on the same line so markdown does the right thing... Update the bumpversion file accordingly, also
```suggestion
### [1.0.0](https://github.com/plus3it/terraform-aws-tardigrade-resource-group/releases/tag/1.0.0)
```